build_tools/bazel_to_cmake/bazel_to_cmake_targets.py

```diff
@@ -122,6 +130,8 @@ def convert_external_target(target):
     return EXPLICIT_TARGET_MAPPING[target]
   if target.startswith("@com_google_absl//absl"):
     return _convert_absl_target(target)
+  if target.startswith("@llvm-project//llvm"):
```

Comment:
you can also remove the explicit mappings near the top of this file

Reply:
Apparently I forgot to commit that. Done.